### PR TITLE
[Fix] Account Delete

### PR DESCRIPTION
**Pull request by gertie-sheshe**
## Description

Issues Fixed:
- Delete button not showing on list accounts. Link to [Pivotal](https://www.pivotaltracker.com/story/show/180935378)
- Delete button spacing in Modal. Link to [Pivotal](https://www.pivotaltracker.com/story/show/180958750)
- Adjust twitter image icon size as per design.

## Type of change

Please delete options that are not relevant.

- [ ] Chore (non-breaking change which does not add visible functionality but improves code quality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Delete Button:
Screenshot 2022-01-19 at 09 07 49

### Spacing:
Screenshot 2022-01-19 at 09 51 32

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

For whatever reason I can't run TwoopsTrackerUI locally. I think it's my machine throwing:
```sh
error - ./src/assets/icons/avator.svg
Error: Cannot find module './node'
Require stack:
- /home/clemence/Work/github/CodeForAfrica/TwoopsTrackerUI/node_modules/@babel/generator/lib/printer.js
...
```

Anyways, don't wait for my 🚀 on this. You can merge once other team members approve.

File: src/components/AccountsList/index.js
Comment (gertie-sheshe) [reply src/components/AccountsList/index.js:39]: This was throwing an error after passing down the props, figured it was because of how the parameters were being passed down to useSWR, and also realized we do not have pagination for this page implemented, I asked @esirK  to look at it, and seems we might have a different data structure for this particular one - so in the meantime I have commented out that line 😅 @kilemensi 

Screenshot 2022-01-19 at 12 50 10

Comment (KhadijaMahanga) [reply src/components/AccountsList/index.js:39]: @gertie-sheshe we need to fix `pagination` on this page, either on this PR or another. This line isn't an issue at all!

We are passing undefined as page & pageProps ...so the string becomes `page=undefined&pageSize=undefined` ...and then in `useSWR` we are not concatenating the URL with `?` ...so the result URL becomes `/api/lists/{id}page=undefined&pageSize=undefined`

Anyways, if we want to hide Pagination for now, we should remove the entire
```
  <Pagination 
     ...
  />
```

**Review by KhadijaMahanga — COMMENTED** [review]
The Delete looks good!

File: src/components/AccountsList/index.js
Comment (gertie-sheshe) [reply src/components/AccountsList/index.js:39]: Yeah, I noticed that and it is fixed I just haven't pushed it because pagination hasn't been implemented on the backend (screenshot above), so even if I pass that it does not fix pagination. @esirK mentioned the structure will be different for this case so still waiting on feedback from him @KhadijaMahanga

Comment (gertie-sheshe) [reply src/components/AccountsList/index.js]: Removed Pagination, to be addressed after it is done on the backend as agreed after the discussion @KhadijaMahanga

**Review by KhadijaMahanga — APPROVED** [review]
🚀

**Pull request merged**
